### DISCUSSION TITLE
Compatible with FMX but only for Windows?

### DISCUSSION THREAD

**Discussion by nosakiis** [landrix/Zint-Barcode-Generator-for-Delphi#23] [Q&A]
The best interest to use FMX is to obtain a multiplatform app.

If you try to compile your Zint-barcode-Generator for another platform than Windows, you cannot do it!!!

Why? Because your Zint-barcode-Generator need the unit... Windows!

It would be interesting to have a really multiplatform solution

**Comment by landrix** [comment]
currently i don't have any time for investigation on this
pull request are welcome
